_data/versions.yml

```diff
@@ -1,17 +1,43 @@
 v3.0:
-- title: v3.0.0
+- title: v3.0.1
   note: |
-    21 December 2017
+    22 December 2017
 
     #### What's new
+
+    > **Important** This release includes breaking changes introduced as part of Calico v3.0.
+    > If you are upgrading a previous Calico deployment to Calico v3.0 you
+    > must follow this one-time migration process as part of the
+    > [upgrade process](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/).
+    > Please carefully review the upgrade preocess before attempting to upgrade
+    > to Calico v3.0.
+    {: .alert .alert-danger}
+
+    ##### Migration and upgrade from v2.6.5
+    
+    - This version of Calico supports [migration and upgrade](/v3.0/getting-started/kubernetes/upgrade/)
+      from Calico v2.6.5. 
+
+    - The Calico resource model has been updated which requires that Calico
+      manifests that are loaded using the `calicoctl` command must be updated or
```

Comment:
I think you need some commas in this sentence or to break it up.
Maybe `The Calico resource model has been updated. Calico manifests, which are loaded using the `calicoctl` command, must be updated or converted to the new model.`  I don't know, words are hard.

Reply:
yep, good catches. I generally hate commas, but didn't fix up the subjunctive clauses correctly :-)

Reply:
@gunjan5 , @tmjd ... how about:
```
- The Calico resource model has been updated. This change requires that Calico 
  manifests must be converted to the new model using the `calicoctl convert` command.

```

Reply:
we should make it clear that it's only the resources created by calicoctl. I'm wondering if people can get confused by "Calico manifests" and try to convert calico.yaml manifest with calicoctl convert. Maybe I'm reading too much into this